sw/airborne/modules/cam_control/cam.c

```diff
@@ -303,13 +302,17 @@ void cam_waypoint_target(void)
   cam_target();
 }
 
+#ifdef TRAFFIC_INFO
+#include "modules/multi/traffic_info.h"
+
 void cam_ac_target(void)
 {
-#ifdef TRAFFIC_INFO
-  struct ac_info_ * ac = get_ac_info(cam_target_ac);
-  cam_target_x = ac->east;
-  cam_target_y = ac->north;
-  cam_target_alt = ac->alt;
+  struct EnuCoor_f ac_pos *ac = acInfoGetPositionEnu_f(cam_target_ac);
+  cam_target_x = ac->x;
+  cam_target_y = ac->y;
+  cam_target_alt = ac->z;
```

Comment:
It looks like cam_target_alt currently expects the alt as hmsl... here you set it to height over local origin.